portal/views/truenth.py

```diff
@@ -177,6 +177,7 @@ def expires_in():
     cookie_timeout = request.cookies.get('SS_TIMEOUT')
 
     disable_links = True if 'disable_links' in request.args else False
+    login_as = True if 'login-as' in session else False
```

Comment:
This could be moved into `render_template()` and simplified as `login='login-as' in session`